docs/plugin-development/restrictions.md

```diff
@@ -159,7 +159,7 @@ experience.
 
 ## How are plugins reviewed and approved?
 
-[This page documents our plugin submission process](plugin-submission), and the
+[This page documents our plugin submission process](https://dalamud.dev/plugin-development/plugin-submission), and the
```

Comment:
This is happening because we're doing things wrong. Docusaurus is supposed to [intelligently format links](https://docusaurus.io/docs/markdown-features/links) written properly, which require the `.md`. 

Done right, this also means we can navigate appropriately in other Markdown-compliant editors or viewers.

```suggestion
[This page documents our plugin submission process](./plugin-submission.md), and the
```